src/PolykeyAgent.ts

```diff
@@ -457,13 +454,13 @@ class PolykeyAgent {
           connectionCallback: (streamPair) =>
             rpcServerClient!.handleStream(streamPair, {}),
           fs,
-          host: _networkConfig.clientHost,
-          port: _networkConfig.clientPort,
+          host: networkConfig_.clientHost,
+          port: networkConfig_.clientPort,
           tlsConfig,
-          maxReadBufferBytes: _networkConfig.maxReadBufferBytes,
-          idleTimeout: _networkConfig.idleTimeout,
-          pingInterval: _networkConfig.pingInterval,
-          pingTimeout: _networkConfig.pingTimeout,
+          maxReadableStreamBytes: networkConfig_.maxReadBufferBytes,
```

Comment:
This stuff should exist inside `src/config.ts`.

Reply:
Any missing config has been added and names have been updated.